### PR TITLE
Move to GitHub Actions

### PR DESCRIPTION
**Pull request by ssanj**
Using GH Actions in preference to Travis CI (which is dead?).

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: CI
+on:
+  push:
+
+jobs:
+  jvm:
+    strategy:
+      fail-fast: true
+      matrix:
+        java: [11, 16]
```
Comment (lukestephenson) [inline .github/workflows/ci.yml:10]: Just java 17 perhaps? 16 wasn't a LTS release

Comment (ssahayam-zd) [reply .github/workflows/ci.yml:10]: Yeah, I tried Java 17 but [AdoptOpenJDK](https://adoptopenjdk.net/) doesn't support it as yet. We could try and use another Java provider?

Comment (ssanj) [reply .github/workflows/ci.yml]: I managed to get `openjdk` working with Java 17.

**Review by wjlow — APPROVED** [review]

**Comment by benhutchison** [comment]
Im a fan of https://github.com/djspiewak/sbt-github-actions myself

basic idea is you write your github actions in build.sbt using a scala dsl, then run `githubWorkflowGenerate`  to emit the yml files, advantage being that your whole build is defined in one file. But I admit its not a huge benefit, Ive just grown into the habit..

**Pull request merged**